cmd/influxd/inspect/dump_tsi/dump_tsi.go

```diff
@@ -0,0 +1,513 @@
+package dump_tsi
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"regexp"
+	"text/tabwriter"
+
+	"github.com/influxdata/influxdb/v2/models"
+	"github.com/influxdata/influxdb/v2/tsdb"
+	"github.com/influxdata/influxdb/v2/tsdb/index/tsi1"
+	"github.com/spf13/cobra"
+)
+
+type args struct {
+	seriesFilePath string
+	paths          []string
+
+	showSeries         bool
+	showMeasurements   bool
+	showTagKeys        bool
+	showTagValues      bool
+	showTagValueSeries bool
+
+	measurementFilter *regexp.Regexp
+	tagKeyFilter      *regexp.Regexp
+	tagValueFilter    *regexp.Regexp
+
+	w *io.Writer
```

Comment:
```suggestion
	w io.Writer
```
Since `io.Writer` is an interface, it can be `nil` without the extra layer of indirection